### DISCUSSION TITLE
Calibration Rejected: No square icon

### DISCUSSION THREAD

**Discussion by parapenT1sta** [NightscoutFoundation/xDrip#3244] [Bug reports]
Can you please clarify how the calibration dot is shown when a calibration is rejected? I thought it would become a square. In my case using Dexcom G5 is shown as a circle even after 2 minutes the calibration was rejected.

![Screenshot_20231218_075428_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/608650/b8e37556-33d2-4399-8c48-b43fdb536e3c)

![Screenshot_20231218_075609_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/608650/94e8bc35-458c-4152-8c7a-4db1927b5b0b)

**Comment by Navid200** [comment]
Are you using G5 in native on non-native mode?

**Reply by parapenT1sta** [reply]
Native Algorithm.

**Reply by Navid200** [reply]
I believe xDrip leaves that circle because it cannot really reject the calibration.  It has already submitted it to the transmitter.  
xDrip itself will reject the calibration and will not use it for non-native values.
But, it's up to the transmitter to accept or reject calibration for native readings.

**Reply by Navid200** [reply]
I'm not up to date with the details of how/when/if a square is changed to a circle.  I thought that if you enter a blood glucose value, it will be shown as a square.  Now, if you choose it and select to use it as a calibration, it may be changed to a circle.

But, if you enter a calibration, I don't know if it will ever be changed to a square.
